SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -541,6 +546,51 @@ void USpatialWorkerConnection::ProcessOutgoingMessages()
 				nullptr);
 			break;
 		}
+		case EOutgoingMessageType::Metrics:
+		{
+			FMetrics* Message = static_cast<FMetrics*>(OutgoingMessage.Get());
+
+			// Do the conversion here so we can store everything on the stack.
```

Comment:
me no likey this :(

Reply:
I just realized we could handle this with a `FMemStack`.